README.md

```diff
@@ -133,7 +134,12 @@ In your `pom.xml` file, add:
 <dependency>
     <groupId>io.quarkiverse.scala</groupId>
     <artifactId>quarkus-scala3</artifactId>
-    <version>0.0.1<version>
+    <version>1.0.0<version>
+</dependency>
+<dependency>
+  <groupId>io.quarkiverse.scala</groupId>
+  <artifactId>quarkus-scala3-deployment</artifactId>
+  <version>1.0.0</version>
```

Comment:
No, you don't need to do that. Why do you think it's needed?

Reply:
removed it